test/close-request.test.js

```diff
@@ -0,0 +1,28 @@
+import t from 'tap'
+import Fastify from 'fastify'
+import FakeTimers from '@sinonjs/fake-timers'
+import slowDownPlugin from '../index.js'
+import { HEADERS } from '../lib/constants.js'
+
+t.test('Check the response time with 2 enpoints', async t => {
+  t.context.clock = FakeTimers.install()
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  fastify.get('/cancel', async () => 'Hello from Fastify!')
+  t.context.clock.tick(1)
+
+  const response = await fastify.inject({
+    path: '/cancel',
+    method: 'GET',
+    simulate: {
+      close: true
+    }
+  })
+
+  t.equal(response.headers[HEADERS.closed], true)
```

Comment:
as mentioned before, we should simply test that we're getting an empty response back, no need to check for a header